config.js

```diff
@@ -1,45 +1,38 @@
-var path = require('path');
-var fs = require('fs');
+const fs = require('fs');
+const path = require('path');
 
 // Set some default configuration
 var server = {
   loggerLevel: "info",
 };
 
 
-function loadConfig(configDirPath) {
-    console.log(`Looking for config files in directory "${path.resolve(configDirPath)}"`);
+function loadConfig(configPath) {
+  configPath = path.resolve(configPath);
+  checkConfigFileExists(configPath);
 
-    checkConfigFileExists(configDirPath, 'app.js');
-    
-    var config = require(configDirPath + '/app');
+  const config = require(configPath);
+  for (var key in config) {
+    exports[key] = config[key];
+  }
 
-    for (var key in config) {
-        exports[key] = config[key];
-    }
+  checkDatabaseConfigAvailable(config);
 
-    testDatabaseConfigAvailable(config);
-
-    return exports;
+  return exports;
 }
 
```

Comment:
Might be worth using path.resolve(configDirPath) to get the full path here for this error.   That way user can see exactly where the program is looking.

Reply:
Already done. The first thing that loadConfig() does is path.resolve(configPath).

Reply:
Sorry missed that.